cmd/goal/application.go

```diff
@@ -17,17 +17,21 @@
 package main
 
 import (
+	"bytes"
+	"crypto/sha512"
 	"encoding/base32"
 	"encoding/base64"
 	"encoding/binary"
 	"fmt"
+	"github.com/algorand/go-algorand/libgoal"
```

Comment:
Should this import be grouped with the other `algorand` imports below?

Reply:
yes